dags/pull_wys.py

```diff
@@ -93,7 +94,11 @@ def custom_fail_slack_alert(context: dict) -> str:
                         'api_key':api_key}
                         )
     
-    t2 = PythonOperator(
+    @task
```

Comment:
You should add the on_failure_callback here

https://stackoverflow.com/a/76718651/4047679

Reply:
Yes you are correct.